modules/ipi-install-configuring-host-network-interfaces-for-subnets.adoc

```diff
@@ -12,9 +12,11 @@ If you have established different network segments or subnets for remote worker
 
 [IMPORTANT]
 ====
-All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.
+In {product-title} {product-version}, you can run control plane nodes in the same subnet or multiple subnets. A multiple subnet environment is useful for reducing the risk of your {product-title} cluster from failing because of a hardware failure or a network outage.
 
-Deploying a cluster with multiple subnets requires using virtual media, such as `redfish-virtualmedia` and `idrac-virtualmedia`.
+For a multiple subnet environment, create a manifest configuration file and specify more than one virtual IP address in the `ingressVIP` parameter of the file. You must also configure an external load balancer in place of the default load balancer. For more information, see "Configuring network components to run on the control plane" and "Services for a user-managed load balancer".
```

Comment:
Multiple ingress VIPs are not necessary here. The ingressVIPs parameter (note that the singular field name is deprecated) should just be the external loadbalancer address. Likewise for apiVIPs.

Reply:
Thanks. Ah, yes, I see this stated elsewhere that those parameters are deprecated. Updated.